apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -91,7 +108,10 @@ type NetworkService interface {
 
 // MachineManagerAPI provides access to the MachineManager API facade.
 type MachineManagerAPI struct {
+	model                   coremodel.ReadOnlyModel
 	controllerConfigService ControllerConfigService
+	bootstrapProvider       providertracker.ProviderGetter[environs.BootstrapEnviron]
+	instanceTypeProvider    providertracker.ProviderGetter[environs.InstanceTypesFetcher]
```

Comment:
as above, replace these with function types so we don't have to import the `providertracker' package.

Reply:
Same comment as above